app/controllers/application_controller.rb

```diff
@@ -73,8 +73,8 @@ def set_last_updated
     # rubocop:enable Naming/MemoizedInstanceVariableName
   end
 
-  def set_enable_service_switch_banner_in_action
-    @enable_service_switch_banner_in_action = true
+  def set_switch_banner
```

Comment:
I know I'm going to seem really picky here - but `switch_banner` doesn't make any reference to what its switching - a potential point of confusion for future developers.

Can this be renamed to something like `service_switch_banner` which is a bit more specific but still short?

Reply:
👍